.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -0,0 +1,28 @@
+name: Maintainer approval for requirement edits
+
+on:
+  pull_request:
```

Comment:
can we run this workflow when a PR is approved?

Reply:
yeah i think that would be better. i originally wanted to avoid running this workflow on every PR approval, so i was trying to see if there's a way to run on "PR review added for PR that touches specified paths". however it doesn't seem possible, so i'll probably have to check the PR paths in the job steps.